testing/src/test/java/org/oppia/android/testing/logging/EventLogSubjectTest.kt

```diff
@@ -0,0 +1,298 @@
+package org.oppia.android.testing.logging
+
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.oppia.android.app.model.AppLanguageSelection
+import org.oppia.android.app.model.AudioTranslationLanguageSelection
+import org.oppia.android.app.model.EventLog
+import org.oppia.android.app.model.EventLog.ExplorationContext
+import org.oppia.android.app.model.EventLog.TopicContext
+import org.oppia.android.app.model.OppiaLanguage
+import org.oppia.android.app.model.ProfileId
+import org.oppia.android.app.model.WrittenTranslationLanguageSelection
+
+@RunWith(JUnit4::class)
+class EventLogSubjectTest {
+  @Test
+  fun testHasTimeStamp_withTimeStamp_matchesTimeStamp() {
+    val eventLog = EventLog.newBuilder()
+      .setTimestamp(123456789)
+      .build()
+
+    EventLogSubject.assertThat(eventLog)
+      .hasTimestampThat()
+      .isEqualTo(123456789)
+  }
+
+  @Test(expected = AssertionError::class)
+  fun testHasTimeStamp_withDifferentTimeStamp_fails() {
+    val eventLog = EventLog.newBuilder()
+      .setTimestamp(123456789)
+      .build()
+
+    EventLogSubject.assertThat(eventLog)
+      .hasTimestampThat()
+      .isEqualTo(987654321)
+  }
```

Comment:
We don't use this pattern when testing. Prefer something likee:
```suggestion
  @Test
  fun testEventLogSubject_failsOnUnmatchingTimestamp() {
    val eventLog = EventLog.newBuilder()
      .setTimestamp(123456789)
      .build()

 assertThrows<AssertionError>() {
      EventLogSubject.assertThat(eventLog)
      .hasTimestampThat()
      .isEqualTo(987654321)
    }   
  }
```